src/states/adult.rs

```diff
@@ -115,13 +115,17 @@ impl Adult {
         Ok(node)
     }
 
+    pub fn closest_elders_to(&self, name: &XorName) -> impl Iterator<Item = &P2pNode> {
```

Comment:
Maybe the name is not quite accurate enough, it does not suggest that we then need to ask these for the closest elders to? (Could it be something like `closest_known_elders_to`? does not seem ideal).